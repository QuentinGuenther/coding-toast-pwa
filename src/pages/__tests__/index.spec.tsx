import * as React from 'react';
import { mount } from 'enzyme';
import HomePage from '../index';

describe('Pages', () => {
  describe('Index', () => {
    it('should render correctly', () => {
      const wrapper = mount(<HomePage />);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
