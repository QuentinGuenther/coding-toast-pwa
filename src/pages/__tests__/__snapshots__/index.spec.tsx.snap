// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Index should render correctly 1`] = `"<nav class=\\"bg-background-primary navbar navbar-dark bg-dark sticky-top fixed-top\\"><span class=\\"navbar-brand\\"><img style=\\"width: 12em;\\" src=\\"/logo.svg\\" alt=\\"coding toast\\" class=\\"\\"></span><button aria-controls=\\"responsive-navbar-nav\\" type=\\"button\\" aria-label=\\"Toggle navigation\\" class=\\"navbar-toggler collapsed\\"><span class=\\"navbar-toggler-icon\\"></span></button><div class=\\"navbar-collapse collapse\\" id=\\"responsive-navbar-nav\\"><div class=\\"text-right navbar-nav\\"><div class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\" role=\\"button\\"><a class=\\"text-text-color-primary\\" href=\\"/blog\\">Blog</a></a></div><div class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\" role=\\"button\\"><a class=\\"text-text-color-primary\\" href=\\"/resources\\">Resources</a></a></div><div class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\" role=\\"button\\"><a class=\\"text-text-color-primary\\" href=\\"/contact\\">Contact</a></a></div></div></div></nav><main><div>Hello Next.js</div></main>"`;
